ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh

```diff
@@ -17,7 +17,17 @@ fi
 ## determine if it is hypershift guest cluster or not
 if ! (oc get node --kubeconfig=${KUBECONFIG} | grep master) ; then
   echo "Testing on hypershift guest cluster"
-  ./console-test-frontend-hypershift.sh || true
+  if (oc get authentication cluster -o=jsonpath={.spec.type} --kubeconfig=${KUBECONFIG} | grep OIDC);then
+    echo "Testing on Cluster using external OIDC"
+    export USER_EMAIL=$(cat /var/run/hypershift-qe-ci-rh-sso-service-account/email)
+    export USER=$(cat /var/run/hypershift-qe-ci-rh-sso-service-account/rh-username)
+    export PASSWORD=$(cat /var/run/hypershift-qe-ci-rh-sso-service-account/rh-password)
+    export issuer_url=$(cat /var/run/hypershift-ext-oidc-cli/issuer-url)
+    export cli_client_id=$(cat /var/run/hypershift-ext-oidc-cli/client-id)
+    ./console-test-azure-external-oidc.sh || true
```

Comment:
Could we allow the propagation of the exit status ?  
```suggestion
    ./console-test-azure-external-oidc.sh
```

Reply:
Also, is it possible to invoke `set -e` somewhere, so `./console-test-azure-external-oidc.sh` failures causes the entire step to fail (and later test steps will not run) ?